cypress/support/commands.js

```diff
@@ -14,8 +14,17 @@ import {
   createCustomType,
   renameCustomType,
   addFieldToCustomType,
+  addSlicesToCustomType,
+  saveCustomTypeModifications,
```

Comment:
Can we (should we?) import and load all the exported functions automatically with a loop?

Reply:
I've never done that before but it looks like it would be nice.